controllers/imageclusterinstall_controller.go

```diff
@@ -831,6 +831,11 @@ func (r *ImageClusterInstallReconciler) ensureCreds(ctx context.Context, log log
 	if err := r.Credentials.EnsureKubeconfigSecret(ctx, log, cd, filepath.Join(workDir, authDir, credentials.Kubeconfig)); err != nil {
 		return fmt.Errorf("failed to ensure kubeconfig secret: %w", err)
 	}
+
+	if err := r.Credentials.EnsureSeedReconfigurationSecret(ctx, log, cd, filepath.Join(workDir, ClusterConfigDir, credentials.SeedReconfigurationFileName)); err != nil {
```

Comment:
Why does this file path have `ClusterConfigDir` in it twice?

`workDir` is already `r.Options.DataDir/namespaces/ici.Namespace/string(ici.ObjectMeta.UID)/files/ClusterConfigDir`

So now we end up with this path being `/data/namespaces/ibi-test/377e0bc8-939f-40ac-859a-826853bbded3/files/cluster-configuration/cluster-configuration/manifest.json`

Was that intentional?